services/ubuntu/ubuntu.service.js

```diff
@@ -0,0 +1,72 @@
+'use strict'
+
+const Joi = require('joi')
+const { renderVersionBadge } = require('../version')
+const { BaseJsonService, NotFound } = require('..')
+
+const schema = Joi.object({
+  entries: Joi.array()
+    .items(
+      Joi.object({
+        source_package_version: Joi.string().required(),
+      })
+    )
+    .required(),
+}).required()
+
+module.exports = class Ubuntu extends BaseJsonService {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'ubuntu/v',
+      pattern: ':series?/:packageName',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Ubuntu package',
+        namedParams: { series: 'bionic', packageName: 'ubuntu-wallpapers' },
+        staticPreview: renderVersionBadge({ version: '18.04.1-0ubuntu1' }),
+      },
+    ]
+  }
+
+  static get defaultBadgeData() {
+    return { label: 'ubuntu' }
+  }
+
+  async handle({ series, packageName }) {
+    const seriesParam = series
+      ? {
+          distro_series: `https://api.launchpad.net/1.0/ubuntu/${encodeURIComponent(
+            series
+          )}`,
+        }
+      : {}
+    const data = await this._requestJson({
```

Comment:
What would you think about including an http 400 error message check to catch the invalid series (similar to what you did in in #3372 at https://github.com/badges/shields/pull/3372/files#diff-339abe4445a00826d7df94d8429ccf9fR49). 

Perhaps the message could be something like `invalid series` or `series not found`?

Reply:
Ah, somehow I was under the impression that the service wouldn't actually respond with 400 for such requests. But it in fact does, thanks for pointing it out! And done.